libs/api/domains/communications/src/lib/communications.service.ts

```diff
@@ -135,6 +135,19 @@ export class CommunicationsService {
       return false
     }
 
+    let recipient = form.recipient
+
+    // The CMS might have a form field which decides who the recipient is
+    if (input.recipientDeciderValue && form.recipientFormFieldDecider) {
+      const recipientAccordingToCMS =
+        form.recipientFormFieldDecider.recipientDecider?.[
+          input.recipientDeciderValue
+        ]
+      if (recipientAccordingToCMS) {
+        recipient = recipientAccordingToCMS
+      }
+    }
+
```

Comment:
Would something like this also work? How does `input.recipientDeciderValue` play into this? I'm just wondering whether we can remove a nesting level without resorting to long ternaries.
```suggestion
    const recipientAccordingToCMS =
        form.recipientFormFieldDecider?.recipientDecider?.[
          input.recipientDeciderValue
        ]
    let recipient = recipientAccordingToCMS ?? form.recipient
```

Reply:
Nice suggestions, I'll change this :)

Reply:
Your suggestion seems to cause a lint error since we are potentially indexing with undefined
![image](https://user-images.githubusercontent.com/43557895/188881697-ab09c394-23b9-417e-b5b7-21850b7bf66a.png)

Reply:
This however works 
![image](https://user-images.githubusercontent.com/43557895/188905000-8f74e6a1-4c75-4fa7-b2d3-5311ee16d339.png)